Library/Formula/mono.rb

```diff
@@ -3,6 +3,7 @@ class Mono < Formula
   homepage "http://www.mono-project.com/"
   url "http://download.mono-project.com/sources/mono/mono-4.0.2.5.tar.bz2"
   sha256 "b074584eea5bbaaf29362486a69d70abe53d0d2feb334f231fa9c841cf6fd651"
+  revision 1
```

Comment:
I don't think it's worth forcing an upgrade on everyone for that edge-case, thanks.